bandwidth-file-spec.txt

```diff
@@ -208,6 +208,66 @@ It consists of:
 
     This Line has been added in version 1.1.0 of this specification.
 
+  "percent_eligible_relays=" Int NL
```

Comment:
This spec might be easier to read if you put the keys that don't depend on other keys first.

What do you think of this structure?

Start a new subsection

number_eligible_relays
  * define eligible

minimum_percent_eligible_relays
  * say what happens when we don't have the minimum percentage

number_consensus_relays
percent_eligible_relays
minimum_number_eligible_relays

Reply:
In this case i should also change what you comment in https://github.com/torproject/torspec/pull/38#discussion_r231435563. I'm fine with that.